### PR TITLE
Allow heads of staff to message Centcom

### PR DESCRIPTION
**Pull request by nfreader**
Or the Syndicate, if that's their jam.

## About The Pull Request
A two line change that allows anyone with head of staff access to transmit a message to central command

## Why It's Good For The Game
There are occasionally times when it may become necessary to go over the captain's head

## Changelog
:cl: Atlanta Ned
add: If you've got Head of Staff access, you can now message Central Command. Central Command is not responsible for any executions sustained as a result of mutinous behavior.
/:cl:

### CODE REVIEW

File: code/game/machinery/computer/communications.dm [hunk 1]
```diff
@@ -278,7 +278,7 @@
 
 		// OMG CENTCOM LETTERHEAD
 		if("MessageCentCom")
-			if(authenticated==2)
+			if(authenticated==1)
```
Comment (SpaceManiac) [inline code/game/machinery/computer/communications.dm]: Seems like this disables it when logged in as Cap, probably just want `if(authenticated)` as elsewhere in the file

**Review by SpaceManiac — CHANGES_REQUESTED** [review]

**Comment by daklaj** [comment]
ert every round lets go

**Comment by zxaber** [comment]
When do we allow AIs to message Centcomm so I can taunt them when I'm Malf?

File: code/game/machinery/computer/communications.dm (hunk 1 above)
Comment (nfreader) [reply code/game/machinery/computer/communications.dm]: I am the big dumb. Thanks for catching this.

**Comment by FloranOtten** [comment]
> ert every round lets go

Only if the admins present agree to it.

**Comment by Potato-Masher** [comment]
I thought this was a nice feature to act as sort of a special privilege to being the Captain that no other person has access to at round start. I mean this is okay, but this just makes Captain less special than it is.

**Comment by Poojawa** [comment]
a station captain really isn't anything special in a codebase that everyone can do everything anyway

**Comment by ATH1909** [comment]
> a station captain really isn't anything special in a codebase that everyone can do everything anyway

Hey, there's some job-exclusive stuff. Off the top of my head, lawyers, centcom officers (IIRC), and HoPs have a higher chance to save the soul of a hellbound person than normal crewmembers do.

**Pull request merged**